site/app/templates/admin/UploadConfigForm.twig

```diff
@@ -35,30 +34,47 @@
     <br>
     <b>located in {{ target_dir }}</b>
     <br>&nbsp;<br>
+    <link>
+    <script>
+        function renameConfig(file_path){
+            var url = buildUrl({'component': 'admin', 'page': 'gradeable', 'action': 'check_being_used', 'config': file_path});
+            $('#alert_in_use').html('');
+            $('ol').empty();
+            $.ajax({
+                url : url,
+                success(data){
+                    var gradeable_ids = JSON.parse(data);
+                    if(gradeable_ids.length > 0){
+                        $('#alert_in_use').html("Note: Currently these gradeables are using this config, rename at your own risk");
```

Comment:
It seems kind of weird to tell the user about this retroactively.  You could probably just word it differently so its more clear that the changes have already been made.

Reply:
That ajax call is only for checking if it's being used or not, it's not actually making any changes.